dbflute-engine/src/main/java/org/dbflute/helper/jdbc/sqlfile/DfSqlFileRunnerBase.java

```diff
@@ -181,9 +182,9 @@ protected InputStreamReader newInputStreamReader() {
             final String encoding = _runInfo.isEncodingNull() ? "UTF-8" : _runInfo.getEncoding();
             return new InputStreamReader(new FileInputStream(_sqlFile), encoding);
         } catch (FileNotFoundException e) {
-            throw new BuildException("The file does not exist: " + _sqlFile, e);
+            throw new IllegalStateException("The file does not exist: " + _sqlFile, e);
```

Comment:
Can we get the same result by IllegalStateException?

Reply:
I just tried it, same result like this:
```
[df-generate] templates
[df-generate] 2018-05-26 23:11:57,518 ERROR - Look! Read the message below.
[df-generate] /* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
[df-generate] Failed to execute DBFlute Task 'Generate'.
[df-generate] 
[df-generate] [Advice]
[df-generate] Check the exception messages and the stack traces.
[df-generate] * * * * * * * * * */
[df-generate] aaaaa
[df-generate] 	at org.apache.torque.task.TorqueDataModelTask.begin(TorqueDataModelTask.java:112)
[df-generate] 	at org.dbflute.task.bs.DfAbstractTexenTask$1.callBegin(DfAbstractTexenTask.java:97)
[df-generate] 	at org.dbflute.task.bs.assistant.DfTaskBasicController.begin(DfTaskBasicController.java:134)
[df-generate] 	at org.dbflute.task.bs.assistant.DfTaskBasicController.execute(DfTaskBasicController.java:68)
[df-generate] 	at org.dbflute.task.bs.DfAbstractTexenTask.execute(DfAbstractTexenTask.java:151)
[df-generate] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:288)
[df-generate] 	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
[df-generate] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[df-generate] 	at java.lang.reflect.Method.invoke(Method.java:498)
[df-generate] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:105)
[df-generate] 	at org.apache.tools.ant.Task.perform(Task.java:348)
[df-generate] 	at org.apache.tools.ant.Target.execute(Target.java:357)
[df-generate] 	at org.apache.tools.ant.Target.performTasks(Target.java:385)
[df-generate] 	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1329)
[df-generate] 	at org.apache.tools.ant.helper.SingleCheckExecutor.executeTargets(SingleCheckExecutor.java:38)
[df-generate] 	at org.apache.tools.ant.Project.executeTargets(Project.java:1181)
[df-generate] 	at org.apache.tools.ant.taskdefs.Ant.execute(Ant.java:416)
[df-generate] 	at org.apache.tools.ant.taskdefs.CallTarget.execute(CallTarget.java:105)
[df-generate] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:288)
[df-generate] 	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
[df-generate] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[df-generate] 	at java.lang.reflect.Method.invoke(Method.java:498)
[df-generate] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:105)
[df-generate] 	at org.apache.tools.ant.Task.perform(Task.java:348)
[df-generate] 	at org.apache.tools.ant.Target.execute(Target.java:357)
[df-generate] 	at org.apache.tools.ant.Target.performTasks(Target.java:385)
[df-generate] 	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1329)
[df-generate] 	at org.apache.tools.ant.Project.executeTarget(Project.java:1298)
[df-generate] 	at org.apache.tools.ant.helper.DefaultExecutor.executeTargets(DefaultExecutor.java:41)
[df-generate] 	at org.apache.tools.ant.Project.executeTargets(Project.java:1181)
[df-generate] 	at org.apache.tools.ant.Main.runBuild(Main.java:698)
[df-generate] 	at org.apache.tools.ant.Main.startAnt(Main.java:199)
[df-generate] 	at org.apache.tools.ant.launch.Launcher.run(Launcher.java:257)
[df-generate] 	at org.apache.tools.ant.launch.Launcher.main(Launcher.java:104)
[df-generate] 2018-05-26 23:11:57,558 INFO  - 
[df-generate] _/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/
[df-generate] [Final Message]: 00m00s045ms *Abort
[df-generate] 
[df-generate]   DBFLUTE_CLIENT: {maihamadb}
[df-generate]     database  = h2
[df-generate]     language  = java
[df-generate]     container = guice
[df-generate]     package   = org.docksidestage.hangar.dbflute
[df-generate] 
[df-generate]   DBFLUTE_ENVIRONMENT_TYPE: {}
[df-generate]     additionalSchema = 
[df-generate]     repsEnvType      = ut
[df-generate]     refreshProject   = $$AutoDetect$$
[df-generate] _/_/_/_/_/_/_/_/_/_/ {Generate}

BUILD FAILED
/Users/jflute/works/dbflute/java8/workspace/dbflute-test-active-hangar/mydbflute/dbflute-1.x/build-torque.xml:167: The following error occurred while executing this line:
/Users/jflute/works/dbflute/java8/workspace/dbflute-test-active-hangar/mydbflute/dbflute-1.x/build-torque.xml:184: org.dbflute.exception.DfDBFluteTaskFailureException: 
/* * * * * * * * * * * * * * * * * * * * * * * * *
Failed to execute the DBFlute task: Generate
Look at the log: console or dbflute.log
* * * * * * * * * */

Total time: 3 seconds
jflute-westend:dbflute_maihamadb jflute$ sh manage.sh 23
/nnnnnnnnnnnnnnnnnnnnnnnnnnn
...Calling the Generate task
nnnnnnnnnn/
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=256m; support was removed in 8.0
Buildfile: ../mydbflute/dbflute-1.x/build-torque.xml

check-use-classpath:

generate:

om-classpath:

om-template:
[df-generate] templates
[df-generate] 2018-05-26 23:16:35,270 ERROR - Look! Read the message below.
[df-generate] /* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
[df-generate] Failed to execute DBFlute Task 'Generate'.
[df-generate] 
[df-generate] [Advice]
[df-generate] Check the exception messages and the stack traces.
[df-generate] * * * * * * * * * */
[df-generate] java.lang.IllegalStateException: aaaaa
[df-generate] 	at org.apache.torque.task.TorqueDataModelTask.begin(TorqueDataModelTask.java:112)
[df-generate] 	at org.dbflute.task.bs.DfAbstractTexenTask$1.callBegin(DfAbstractTexenTask.java:97)
[df-generate] 	at org.dbflute.task.bs.assistant.DfTaskBasicController.begin(DfTaskBasicController.java:134)
[df-generate] 	at org.dbflute.task.bs.assistant.DfTaskBasicController.execute(DfTaskBasicController.java:68)
[df-generate] 	at org.dbflute.task.bs.DfAbstractTexenTask.execute(DfAbstractTexenTask.java:151)
[df-generate] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:288)
[df-generate] 	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
[df-generate] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[df-generate] 	at java.lang.reflect.Method.invoke(Method.java:498)
[df-generate] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:105)
[df-generate] 	at org.apache.tools.ant.Task.perform(Task.java:348)
[df-generate] 	at org.apache.tools.ant.Target.execute(Target.java:357)
[df-generate] 	at org.apache.tools.ant.Target.performTasks(Target.java:385)
[df-generate] 	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1329)
[df-generate] 	at org.apache.tools.ant.helper.SingleCheckExecutor.executeTargets(SingleCheckExecutor.java:38)
[df-generate] 	at org.apache.tools.ant.Project.executeTargets(Project.java:1181)
[df-generate] 	at org.apache.tools.ant.taskdefs.Ant.execute(Ant.java:416)
[df-generate] 	at org.apache.tools.ant.taskdefs.CallTarget.execute(CallTarget.java:105)
[df-generate] 	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:288)
[df-generate] 	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
[df-generate] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[df-generate] 	at java.lang.reflect.Method.invoke(Method.java:498)
[df-generate] 	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:105)
[df-generate] 	at org.apache.tools.ant.Task.perform(Task.java:348)
[df-generate] 	at org.apache.tools.ant.Target.execute(Target.java:357)
[df-generate] 	at org.apache.tools.ant.Target.performTasks(Target.java:385)
[df-generate] 	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1329)
[df-generate] 	at org.apache.tools.ant.Project.executeTarget(Project.java:1298)
[df-generate] 	at org.apache.tools.ant.helper.DefaultExecutor.executeTargets(DefaultExecutor.java:41)
[df-generate] 	at org.apache.tools.ant.Project.executeTargets(Project.java:1181)
[df-generate] 	at org.apache.tools.ant.Main.runBuild(Main.java:698)
[df-generate] 	at org.apache.tools.ant.Main.startAnt(Main.java:199)
[df-generate] 	at org.apache.tools.ant.launch.Launcher.run(Launcher.java:257)
[df-generate] 	at org.apache.tools.ant.launch.Launcher.main(Launcher.java:104)
[df-generate] 2018-05-26 23:16:35,332 INFO  - 
[df-generate] _/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/_/
[df-generate] [Final Message]: 00m00s061ms *Abort
[df-generate] 
[df-generate]   DBFLUTE_CLIENT: {maihamadb}
[df-generate]     database  = h2
[df-generate]     language  = java
[df-generate]     container = guice
[df-generate]     package   = org.docksidestage.hangar.dbflute
[df-generate] 
[df-generate]   DBFLUTE_ENVIRONMENT_TYPE: {}
[df-generate]     additionalSchema = 
[df-generate]     repsEnvType      = ut
[df-generate]     refreshProject   = $$AutoDetect$$
[df-generate] _/_/_/_/_/_/_/_/_/_/ {Generate}

BUILD FAILED
/Users/jflute/works/dbflute/java8/workspace/dbflute-test-active-hangar/mydbflute/dbflute-1.x/build-torque.xml:167: The following error occurred while executing this line:
/Users/jflute/works/dbflute/java8/workspace/dbflute-test-active-hangar/mydbflute/dbflute-1.x/build-torque.xml:184: org.dbflute.exception.DfDBFluteTaskFailureException: 
/* * * * * * * * * * * * * * * * * * * * * * * * *
Failed to execute the DBFlute task: Generate
Look at the log: console or dbflute.log
* * * * * * * * * */
```

Reply:
Make DfAntBuildException and use it
to find BuildException point after modification, just in case.